doc/contribute_docs.md

```diff
@@ -0,0 +1,111 @@
+# Contributing to z390 Documentation
+
+The following document will provide details on developing z390 documentation.
+
+It is expected that as part of contributing to the code base, that you also update any documentation impacted by the change.
+
+All documentation will be held in the `doc` folder and be in markdown format.
+Markdown files have a `.md` extension.
+Markdown uses simple text files with specific syntax for formatting. See [the markdown guide](https://www.markdownguide.org/basic-syntax/) for syntax.
+
+You can also use functions provided by the admonition extension in mkdocs. See the [doco for admonition](https://python-markdown.github.io/extensions/admonition/)
+
+## Conventions
+
+### Product names
+
+Product names should always use a small z:
+
+* z390
+* zCOBOL  - Always capitalize COBOL
+* zVSAM   - Always capitalize VSAM
+* zCICS   - Always capitalize CICS
+
+###  Program names and scripts
+
+When specifying tool and script names, always use lower case to be compatible with case sensitive environments like linux.
+
+* mz390
+* az390
+* asmlg
+
+### Units of measurement
+
+For Bytes:
+
+* When specifying a quantity of 1024 bytes, use {quantity}K. Bytes is assumed. Capital K is deliberate.
+* the word bytes can be added if the context does not make it clear that bytes are implied. {quantity}K bytes.
+
+For seconds:
+
+* ms = millisecond
+* &micro;s = microseconds (in markdown &micro; = `&micro;`)
+* ns = nanoseconds
+
+### Code
+
+HLASM code should be enclosed in a code block as follows:
+
+    ``` hlasm
+    LABEL    MVC  X(10),Y
+    ```
+
+### Terminology
+
+The following provides guidelines on the use of specific terminology. If not here, you should use the general 
+principle of following conventions set in IBM mainframe manuals.
+
+term      | usage
+----------|------
+word      | OK. Alternate is fullword with no space.
+halfword  | OK. No space should be used.
+save area | In any text, ensure space is included. The only exception is in code `savearea` without space is preferred 
+ASCII     | Always in capitals
+EBCDIC    | Always in capitals
+
+## Preview the web pages
```

Comment:
OK.. so I looked into the issue where the end of tables was too close to the next line. This was an issue with the html generated from the markdown rather than the actual markdown. I did manage to fix it with the current theme, but I found the skinny format did not really suit the content and the tables all had scroll bars to see the content. I decided instead to move to the material theme which displays the content much better including the tables. Let me know your thoughts on the new site theme. There are a few to choose from.